contract/src/main.rs

```diff
@@ -0,0 +1,128 @@
+#![no_main]
+#![allow(unused_imports)]
+#![allow(unused_parens)]
+#![allow(non_snake_case)]
+
+extern crate alloc;
+
+use alloc::{
+    collections::{BTreeMap, BTreeSet},
+    string::String,
+};
+use core::convert::TryInto;
+
+use casperlabs_contract_macro::{casperlabs_constructor, casperlabs_contract, casperlabs_method};
+use contract::{
+    contract_api::{runtime, storage},
+    unwrap_or_revert::UnwrapOrRevert,
+};
+use types::{
+    account::AccountHash,
+    bytesrepr::{FromBytes, ToBytes},
+    contracts::{EntryPoint, EntryPointAccess, EntryPointType, EntryPoints},
+    runtime_args, CLType, CLTyped, CLValue, Group, Parameter, RuntimeArgs, URef, U256,
+};
+
+#[casperlabs_contract]
+mod ERC20 {
+
+    #[casperlabs_constructor]
+    fn constructor(tokenName: String, tokenSymbol: String, tokenTotalSupply: U256) {
+        set_key("_name", tokenName);
+        set_key("_symbol", tokenSymbol);
+        set_key("_decimals", 18u8);
+        let balances_key = format!("_balances_{}", runtime::get_caller());
+        set_key(&balances_key, tokenTotalSupply);
+        set_key("_totalSupply", tokenTotalSupply);
+    }
+
+    #[casperlabs_method]
+    fn name() {
+        ret(get_key::<String>("_name"));
+    }
+
+    #[casperlabs_method]
+    fn symbol() {
```

Comment:
Remove the `ret` and return normaly. I can also remove `<String>` if you have return type defined.